apps/web/components/ui/form/LocationSelect.tsx

```diff
@@ -40,6 +40,7 @@ export default function LocationSelect(props: Props<LocationOption, false, Group
   return (
     <Select<LocationOption>
       name="location"
+      id="location-select"
```

Comment:
Does `data-testid` not work? We should use a test-specific attribute to target elements in tests.

Reply:
tried using `data-testid` but it's not passed on to the container element.

Reply:
Cool. It;s fine then I guess.